src/classes/groundplaces.ts

```diff
@@ -96,15 +96,20 @@ export class GroundPlacesController {
     const isFilterByStopGroupActive: boolean = filters?.includes(AutocompleteFilter.STOP_GROUP);
     const isFilterByStopClusterActive: boolean = filters?.includes(AutocompleteFilter.STOP_CLUSTER);
     const isFilterByServicedActive: boolean = filters?.includes(AutocompleteFilter.SERVICED);
-    const isFilterWithChildrenActive: boolean = filters?.includes(AutocompleteFilter.SEGMENT_PROVIDER_STOP);
 
     return groundPlaces.filter(
       (place: GroundPlace): GroundPlace => {
         // Checking that the place matching the search query
         if (
           !place.gpuid.toLowerCase().includes(currentQuery) &&
           !place.name.toLowerCase().includes(currentQuery) &&
-          !(place.type === GroundPlaceType.CLUSTER && place.unique_name.toLowerCase().includes(currentQuery))
+          !(place.type === GroundPlaceType.CLUSTER && place.unique_name.toLowerCase().includes(currentQuery)) &&
```

Comment:
Without having any context about these conditions, first feeling is there are a lot of conditions in this block.

Reply:
i think it's quite clear imho